docs/dev-docker/README.md

```diff
@@ -126,93 +122,94 @@ docker run -it --rm --ipc=host --network=host --group-add render \
     rocm/vllm:latest
 ```
 
-Note: The instructions in this document use `/data` to store the models.  If you choose a different directory, you will also need to make that change to the host volume mount when launching the docker container.  For example, `-v /home/username/models:/data` in place of `-v /data:/data` would store the models in /home/username/models on the host.  Some models can be quite large; please ensure that you have sufficient disk space prior to downloading the model.  Since the model download may take a long time, you can use `tmux` or `screen` to avoid getting disconnected.
+> **Note:** This document uses `/data` to store models.  If you choose a different directory, you will also need to update the host volume mount when launching the Docker container.  For example, `-v /home/username/models:/data` instead of `-v /data:/data` will store models in `/home/username/models` on the host. Some models can be large, so ensure you have sufficient disk space before downloading. Because downloads may take a long time, consider using `tmux` or `screen` to prevent disconnection.
 
-### Downloading models with huggingface-cli
+### Downloading models with Hugging Face CLI
```

Comment:
We need to reorder/condense this so that the /data, Downloading, and quantizing your own models sections are all minimal/moved to the end. None of these steps are actually required to run the benchmarks, so the benchmarking steps should be right at the top after the actual perf tables.